bin/concurrency/summary_to_requests_per_second.rb

```diff
@@ -29,7 +29,7 @@
 puts "# offset (ms), req/s"
 start_time.step(end_time, step).each do |d|
   num_active = requests.select do |r|
-    r["end"] > d and r["end"] < (d + step)
+    r["start"] > d and r["start"] < (d + step)
```

Comment:
I think this changes the semantics slightly to be requests made per second rather than requests completed per second. I think we are more interested in completed requests?